documentation/tut/toy-example-expanded.rst

```diff
@@ -103,4 +103,6 @@ First, during the sunny hours of the day, when solar power is being send to the
 
 Second, charging of the battery is also changed a bit (around 10am), as less can be discharged later.
 
-We hope this part of the tutorial shows how to incorporate a limited grid connection rather easily with FlexMeasures. There are more ways to model such settings, but this is a straightforward one.
\ No newline at end of file
+We hope this part of the tutorial shows how to incorporate a limited grid connection rather easily with FlexMeasures. There are more ways to model such settings, but this is a straightforward one.
+
+This tutorial showed the fastest way to a schedule. In :ref:`_tut_toy_schedule_process`, we'll go further into settings with more realistic ingredients: solar panels and a limited grid connection.
```

Comment:
The last part ("go further into settings with more realistic ingredients: solar panels and a limited grid connection") seems to be a copy/paste residue.

You wanted to say something like: "turn to something different: the optimal timing of processes with fixed energy work and duration".

Also re-write this part, actually: "the fastest way to a schedule". Maybe "a quick way to add an inflexible load (like solar power) and a grid connection."

Finally, when I built the docs, the reference "_tut_toy_schedule_process" did not create a link.